docs/source/conf.py

```diff
@@ -1,28 +1,25 @@
 # -*- coding: utf-8 -*-
 # alot documentation build configuration file
 
-import sys, os
+import sys
+import os
 
+from alot import __version__,__author__
```

Comment:
Yes and at the place where it was before there was a modification to sys.path just one line above. The relevant directory to find the local version of alot was added but it is probably not in sys.path up here.